karapace/backup/api.py

```diff
@@ -404,6 +430,9 @@ def create_backup(
         if not records are received within that time and the target offset has not
         been reached an exception is raised. Defaults to one minute.
     :param overwrite: the output file if it exists.
+    :param replication_factor: if specified, metadata will contain this value for
```

Comment:
Could remove the note of `instead of the one...` as the code does not attempt to read the replication factor for the topic.
This was discussed in other media and decision was to not to implement replication factor reading from Kafka. During rebalancing or upgrading the number of partitions can be more than configured. Also there is no good API to retrieve the value from Kafka.